src/applications/mhv/secure-messaging/components/MessageDetailBlock.jsx

```diff
@@ -59,6 +60,10 @@ const MessageDetailBlock = props => {
     [location.pathname, dispatch],
   );
 
+  useEffect(() => {
+    focusElement(document.querySelector('h2'));
+  });
+
```

Comment:
I am curious if this h2 header for message subject should actually be an h1 instead? I am referring to  the element itself, rather then the focus reference. Just thinking that in this case, this page would not have a valid h1 then if this message subject remains h2